lms/djangoapps/courseware/access_utils.py

```diff
@@ -8,17 +8,15 @@
 from logging import getLogger
 
 from django.conf import settings
-from django.utils.translation import ugettext as _
 from pytz import UTC
+
 from lms.djangoapps.courseware.access_response import (
     AccessResponse,
     StartDateError,
     EnrollmentRequiredAccessError,
     AuthenticationRequiredAccessError,
 )
 from lms.djangoapps.courseware.masquerade import get_course_masquerade, is_masquerading_as_student
-from openedx.core.djangoapps.util.user_messages import PageLevelMessages
-from openedx.core.djangolib.markup import HTML
```

Comment:
All changes in this file seem unrelated?

Do you have an auto-linter enabled in your text editor?

Reply:
Sorry there were changes in this file, but they were refactored out. I think is_course_public was here. Will remove.